apps/monitoring/serviceaccount/aat.yaml

```diff
@@ -4,4 +4,5 @@ metadata:
   name: ${NAMESPACE}
   namespace: ${NAMESPACE}
   annotations:
-    azure.workload.identity/client-id: "fe91170e-e2a5-421a-90d0-d7fefab8382c"
\ No newline at end of file
+    azure.workload.identity/client-id: "fe91170e-e2a5-421a-90d0-d7fefab8382c"
+    azure.workload.identity/use: "true"
```

Comment:
Dont think you need this here.
[Chart library](https://github.com/search?q=repo%3Ahmcts%2Fchart-library%20azure.workload.identity%2Fuse&type=code) shows that annotation/label is only use on the pod template